src/read_write_stats.hpp

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2019, ZIH,
```

Comment:
2020 - even though I wish it was still 2019

Reply:
But this code is from 2019 😢

Reply:
oh. ok fine.